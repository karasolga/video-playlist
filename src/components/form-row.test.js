import React from "react";
import FormRow from "./form-row";

describe("FormRow", () => {
  it("renders correctly", () => {
    expect(FormRow).toMatchSnapshot();
  });
});
