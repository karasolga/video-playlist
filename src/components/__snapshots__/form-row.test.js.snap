// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormRow renders correctly 1`] = `[Function]`;
